web/frontend/tests/json/evoting/forms/canceled.json

```diff
@@ -42,6 +42,43 @@
         ],
         "Ranks": [],
         "Texts": []
+      },
+      {
+        "ID": "1NqhDffw",
+        "Title": {
+          "En": "Colours",
+          "Fr": "Couleurs",
+          "De": "Farben"
+        },
+        "Order": [
+          "riJFjw0q"
+        ],
+        "Subjects": [],
+        "Selects": [
+          {
+            "ID": "riJFjw0q",
+            "Title": {
+              "En": "CMYK",
+              "Fr": "CMJN",
+              "De": "CMYK"
```

Comment:
Joke: Wir haben keine Übersetzung dafür? Sowas... "CMGS"?

Reply:
https://www.din.de/de/wdc-beuth:din21:5463877

jedenfalls nicht in Deutschland :woman_shrugging: